ledger/ledger-api-auth/src/main/scala/com/digitalasset/ledger/api/auth/AuthorizationError.scala

```diff
@@ -34,6 +34,12 @@ object AuthorizationError {
       s"Claims are only valid for participantId '$authorized', actual participantId is '$actual'"
   }
 
+  final case class InvalidTargetAudience(actual: List[String], authorized: String)
+      extends AuthorizationError {
+    override val reason =
+      s"Claims are only intended for '$authorized', actual audiences are [${actual.mkString(",")}]"
```

Comment:
Is this error filtered out on the gRPC? I would consider the `$authorized` to be sensitive.
Also I would suggest rephrasing to
```suggestion
      s"Actual audiences [${actual.mkString(",")}] do not match the expected '$authorized'"
```

Reply:
what do you mean by "sensitive" is there anything i have to do to filter it out?

Reply:
I meant the translation of that error to the self-service error code must not reveal any sensitive information on the gRPC. But I have checked and all auth errors get translated in a detail hiding fashion.